sesman/chansrv/clipboard.c

```diff
@@ -372,6 +378,16 @@ clipboard_init(void)
         return 0;
     }
 
+    if (0 != config_read(&g_cfg)) {
```

Comment:
I don't think it's a good design to make xrdp-chansrv read the configuration file of xrdp-sesman.

There is an existing mechanism for disabling channels. The configuration is in `xrdp.ini` and it's xrdp that takes care of it. xrdp is talking to xrdp-chansrv using `server_send_to_channel()`

I would rather make xrdp tell xrdp-chansrv not to send clipboard data to the clients. The configuration parsing would be done in xrdp. The configuration would be in `xrdp.ini`.